modules/ROOT/pages/dw-coercions-functions-todate.adoc

```diff
@@ -6,6 +6,54 @@ endif::[]
 
```

Comment:
For all the new functions in green, we need to verify that these should be added to the docs.

Reply:
It's strange bc some (all?) of these new ones seem to be in the docs already: https://docs.mulesoft.com/dataweave/latest/dw-coercions